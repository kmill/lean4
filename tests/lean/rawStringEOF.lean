#check r###"this is a raw string, unterminated"##
