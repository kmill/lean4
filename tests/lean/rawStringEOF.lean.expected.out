rawStringEOF.lean:1:7: error: unterminated raw string literal
